src/Todo.js

```diff
@@ -1,18 +1,27 @@
 // src/Todo.js
 import React from 'react';
+import crossImg from './x-mark-16.png';
+import { removeTodo, Status } from './todoSlice';
+
+const Todo = ({todo, dispatch }) => {
+
+  const handleDelete = () =>{
+    dispatch(removeTodo(todo.id));
+  }
 
-const Todo = ({ todo, onToggle }) => {
   return (
     <div>
       <input
         type="checkbox"
         style={{height:'25px',width:'25px'}}
         checked={todo.completed}
-        onChange={() => onToggle({type:'complete',id:todo.id})}
+        onChange={() => dispatch(Status(todo.id))}
```

Comment:
you have removeTodo, addTodo, let's have consistent naming. Kinda changeStatusTodo. Moreover upper case is used in Typescript for types